packages/starknet-snap/src/utils/starknetUtils.ts

```diff
@@ -481,3 +486,22 @@ export const validateAndParseAddress = (address: num.BigNumberish, length = 63)
   if (trimmedAddress.length !== length) throw new Error(`Address ${address} has an invalid length`);
   return _validateAndParseAddressFn(address);
 };
+
+export const signTransactions = async (
+  privateKey: string,
+  transactions: Call[],
+  transactionsDetail: InvocationsSignerDetails,
+  abis: Abi[],
+): Promise<Signature> => {
+  const signer = new Signer(privateKey);
+
+  try {
```

Comment:
optional , try catch can be skip for utils function when it has catch at the snap method